Makefile

```diff
@@ -89,10 +91,19 @@ test-coverage:
 	@ ./scripts/coverage.sh
 
 .PHONY: test-style
-test-style:
+test-style: lint
 	GO111MODULE=on golangci-lint run
 	@scripts/validate-license.sh
 
+.PHONY: lint
+lint:
+	set -euo pipefail
+	cd /
+	GO111MODULE=on
+	curl -sSL https://github.com/golangci/golangci-lint/releases/download/v${GOLANGCI_LINT_VERSION}/golangci-lint-${GOLANGCI_LINT_VERSION}-linux-amd64.tar.gz | tar xz
+	sudo mv golangci-lint-${GOLANGCI_LINT_VERSION}-linux-amd64/golangci-lint /usr/local/bin/golangci-lint
+	rm -rf golangci-lint-${GOLANGCI_LINT_VERSION}-linux-amd64
```

Comment:
Might I suggest an alternative method to install it. See https://github.com/Masterminds/semver/blob/master/Makefile where it doesn't use a `.PHONY` target but detects the presence and installs if missing. If present it does not install.

In CircleCI it made sense to install to `/usr/local/bin` but I wouldn't want to do that on every developers system. Can we instead use the GOPATH/bin instead.